src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import duke.command.Command;
+
+/** A bot to help you to keep track of your tasks in a to-do list. */
```

Comment:
Good job with the descriptions for all the classes, they are really easy to understand